bin/spiced/src/tracing.rs

```diff
@@ -173,11 +182,23 @@ fn zipkin_task_history_otel_exporter(
         opentelemetry_zipkin::new_pipeline()
             .with_service_name(service_name)
             .with_collector_endpoint(zipkin_endpoint)
-            .with_http_client(reqwest::Client::new())
+            .with_http_client(Client::new())
             .init_exporter()?,
     )))
 }
 
+async fn is_zipkin_endpoint_reachable(endpoint: &str) -> bool {
+    let client = Client::new();
+    let timeout = Duration::from_secs(5);
+
+    let url = format!("{endpoint}?serviceName=test");
```

Comment:
This will return an empty array and is a good test to ensure we are connected to a valid zipkin endpoint: https://zipkin.io/zipkin-api/#/default/get_spans